### ISSUE TITLE
Typescript types?

### ISSUE THREAD

**Issue by frederikhors** [jacobmischka/svelte-flatpickr#28]
Why this error in a Svelte Typescript project?

```
Could not find a declaration file for module 'svelte-flatpickr'. 'C:/project/node_modules/svelte-flatpickr/dist/index.cjs.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/svelte-flatpickr` if it exists or add a new declaration (.d.ts) file containing `declare module 'svelte-flatpickr';` (ts)
```

**Comment by jacobmischka** [comment]
That's an extremely common warning that happens with any dependency that doesn't have types. Please feel free to contribute types to the @types project as indicated in the warning if you'd like to resolve it. Or just ignore the warning like I always do for untyped deps.

**Comment by frederikhors** [comment]
You don't want to add them to this project?

**Comment by jacobmischka** [comment]
Sure, feel free to submit a .d.ts if you'd like, that's better I suppose.

**Comment by jacobmischka** [comment]
I started this here: https://github.com/jacobmischka/svelte-flatpickr/tree/add-types

Would appreciate anyone more familiar with external type definitions and/or svelte typings to take a look!

**Comment by jacobmischka** [comment]
Fixed in 200fcc7ec2b6c573de4796c9a155516f640f9fe7, published in `3.2.0`. Related issues and suggestions expected and welcome!
